README.md

```diff
@@ -46,7 +43,7 @@ Dependencies
 ------------
 
 - Ansible version > 2.0
-- Beats versions 1.x
+- Beats version > 1.x
```

Comment:
Should be Beats version >= 1.x. Though I hope no one is still running beats 1.X :D